Tests/AppTests/PackageController+routesTests.swift

```diff
@@ -504,6 +504,46 @@ class PackageController_routesTests: AppTestCase {
         }
     }
 
+    func test_defaultTutorial() throws {
+        // setup
+        Current.fetchDocumentation = { _, uri in
+                .init(status: .notFound)
+        }
+        let pkg = try savePackage(on: app.db, "1")
+        try Repository(package: pkg, name: "package", owner: "owner")
+            .save(on: app.db).wait()
+        try Version(package: pkg,
+                    commit: "0123456789",
+                    commitDate: .t0,
+                    docArchives: [.init(name: "docs", title: "Docs")],
+                    latest: .defaultBranch,
+                    packageName: "pkg",
+                    reference: .branch("main"))
+        .save(on: app.db).wait()
+        try Version(package: pkg,
+                    commit: "9876543210",
+                    commitDate: .t0,
+                    docArchives: [.init(name: "docs", title: "Docs")],
+                    latest: .release,
+                    packageName: "pkg",
+                    reference: .tag(1, 0, 0))
+        .save(on: app.db).wait()
+
+        // MUT
+        try app.test(.GET, "/owner/package/tutorials") {
+            XCTAssertEqual($0.status, .notFound)
```

Comment:
I've remove the no-catchall (`**`) route again, because all it'd do is generate a redirect to a 404 page, because there is currently nowhere to link to. We can't point this page to a default tutorial and there's no tutorials overview/list page (yet).

Stream have created one for their project but it doesn't seem to be URL that's available by default: https://swiftpackageindex.com/getstream/effects-library/tutorials/tutorial-table